src/chronify/sqlalchemy/functions.py

```diff
@@ -17,7 +17,7 @@ def read_database(
     """Read a database query into a Pandas DataFrame."""
     df = pl.read_database(query, connection=conn).to_pandas()
     if isinstance(config, DatetimeRange):
-        df = _convert_database_output_for_datetime(df, conn, config)
+        _convert_database_output_for_datetime(df, conn, config)
```

Comment:
df =, no? I don't love the df modification in place.

Reply:
The code was already modifying in place, so this is a correction to reflect reality. For reads, we definitely want to modify in place to avoid extra copies. Writes are debatable. Refer to the docstring on line 32. I could be persuaded to change.

Reply:
My understanding is df modifying in place is ambiguous and inconsistent. I would avoid it.